apps/yapms/src/lib/components/modals/sharemodal/ShareModal.svelte

```diff
@@ -99,13 +102,32 @@
 	function onTurnstileExpired() {
 		turnstileToken = null;
 	}
+
+	async function screenshot() {
+		const mapChart = document.getElementById('map-chart-div');
+		if (mapChart === null) {
+			return;
+		}
+
+		const data = await htmlToImage.toBlob(mapChart, {
+			backgroundColor: 'hsl(var(--b1))'
```

Comment:
This does not work for me. Can we make map-chart-div have this as its background? Do we make it transparent?

Reply:
What does it look like for you?

Reply:
I did your suggestion and it works on chrome now. Submitted a bug report with html-to-image to see if it can work on chrome.